src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -296,42 +300,65 @@ const ActiveState = ({
                                 isFocused={isFocused}
```

Comment:
Adding 

`isHoverable
  hasNewThreadedReplies={hasNewThreadedReplies}
`
Should fix the spacing issues you've been seeing. Should also put the sidebar in a good state for other devs to look at.

Reply:
Yep, totally works. Thanks for that. This will definitely set a better stage for the upcoming UI tasks :)